forms-flow-api/src/formsflow_api/resources/form_process_mapper.py

```diff
@@ -342,3 +352,37 @@ def get(application_id: int):
         except BusinessException as err:
             current_app.logger.warning(err.error)
             return err.error, err.status_code
+
+
+@cors_preflight("POST,OPTIONS")
+@API.route("/form-create", methods=["POST", "OPTIONS"])
+class FormioFormResource(Resource):
+    """Resource for formio form creation."""
+
+    @staticmethod
+    @auth.require
+    @profiletime
+    def post():
+        """Formio form creation method."""
+        try:
+            access_token = cache.get("access_token")
+            if access_token is None:
```

Comment:
When the token expires, cache will clear automatically?

Reply:
yes